fastapi/responses.py

```diff
@@ -21,3 +22,10 @@ class ORJSONResponse(JSONResponse):
     def render(self, content: Any) -> bytes:
         assert orjson is not None, "orjson must be installed to use ORJSONResponse"
         return orjson.dumps(content)
+
+
+class PydanticResponse(JSONResponse):
```

Comment:
Why not name this class ModelResponse ?

Since it's response_model= ... and not response_pydantic= in the definition of the endpoint.

Reply:
Fair point.